### PR TITLE
Admin side PO edit

### PR DESCRIPTION
**Pull request by JasonBaoZ**
Categories from previous PR populated as check boxes in edit profile. Added the page for admin to edit PO profiles. Refactored profile to be its own component.

screen shot 2018-03-16 at 5 57 09 am
screen shot 2018-03-16 at 5 56 50 am
screen shot 2018-03-16 at 5 56 40 am
screen shot 2018-03-16 at 5 56 35 am

### CODE REVIEW

File: app/controllers/partnering_organizations_controller.rb [hunk 1]
```diff
@@ -52,6 +51,11 @@ def create
   # PATCH/PUT /partnering_organizations/1.json
   def update
     respond_to do |format|
+      PartnerCategory.where(partnering_organization_id: @partnering_organization.id).delete_all
```
Comment (JasonBaoZ) [inline app/controllers/partnering_organizations_controller.rb]: Not sure if this is the best way to do it @lbkchen

File: app/controllers/partnering_organizations_controller.rb [hunk 2]
```diff
@@ -52,6 +51,11 @@ def create
...
+      params[:category_ids].each do |i|
+        @po_cat = PartnerCategory.new(partnering_organization_id: @partnering_organization.id, category_id: i)
+        @po_cat.save
```
Comment (JasonBaoZ) [inline app/controllers/partnering_organizations_controller.rb]: Also not sure if theres a way to do these saves at once? @lbkchen

File: app/controllers/partnering_organizations_controller.rb (hunk 1 above)
Comment (lbkchen) [reply app/controllers/partnering_organizations_controller.rb]: This one seems ok

File: app/controllers/partnering_organizations_controller.rb (hunk 2 above)
Comment (lbkchen) [reply app/controllers/partnering_organizations_controller.rb]: Don’t think so :/ u can use create instead of new and save though

**Pull request merged**